### DISCUSSION TITLE
Multiplication of MultivariateNormal with vector

### DISCUSSION THREAD

**Discussion by pabloprf** [cornellius-gp/gpytorch#1681] [Q&A]
The `MultivariateNormal` distribution allows to pass a mean in the form of a matrix to represent a batch of multivariate normals (as indicated in the class description).  The `__mul__` method, however, only allows multiplication of the distribution by a scalar.
I'm interested in multiplying each of the multivariate normals in the batch by a different scalar.
How would you do that?

**Comment by Balandat** [comment]
You'd basically just have to extend the `__mul__` method to also accept a tensor as the `other` argument, and then make sure to properly apply the broadcasted (across batch dims) multiplication to the mean as well as the lazy covariance matrix.

**Reply by Balandat** [reply]
happy to review a PR for this.

**Reply by pabloprf** [reply]
Thanks @Balandat. I will work on it and submit a PR
